torchgeo/datasets/landsat.py

```diff
@@ -207,23 +198,11 @@ class Landsat8(Landsat):
 
     filename_glob = "LC08_*_{}.*"
 
-    default_bands = [
-        "SR_B1",
-        "SR_B2",
-        "SR_B3",
-        "SR_B4",
-        "SR_B5",
-        "SR_B6",
-        "SR_B7",
-        "SR_B8",
-        "SR_B9",
-        "SR_B10",
-        "SR_B11",
-    ]
+    default_bands = ["SR_B1", "SR_B2", "SR_B3", "SR_B4", "SR_B5", "SR_B6", "SR_B7"]
     rgb_bands = ["SR_B4", "SR_B3", "SR_B2"]
 
 
 class Landsat9(Landsat8):
-    """Landsat 9 Operational Land Imager (OLI) and Thermal Infrared Sensor (TIRS)."""
+    """Landsat 9 Operational Land Imager (OLI-2) and Thermal Infrared Sensor (TIRS-2)."""  # noqa: E501
```

Comment:
Black and pydocstyle require one-line docstrings to have the closing quotes on the same line, but flake8 complains that the line is too long. So we have to ignore the latter in order to appease the former.